app/imports/ui/components/BottomHalf.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { Header, Grid, Container } from 'semantic-ui-react';
+
+/** A simple static component to render some text for the landing page. */
+class BottomHalf extends React.Component {
```

Comment:
BottomHalf is a rather confusing component name imho. I'd call LandingInfoText or something similar to indicate that it's a component for the landing page.